client/ratings_reviews/components/ReviewFormModal.jsx

```diff
@@ -61,80 +81,120 @@ class ReviewFormModal extends React.Component {
         name: '',
```

Comment:
it would be good to eventually move the .ajax() to an external file to make this a little less cluttered/better separated